invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -450,12 +451,13 @@ def deposit_edit(pid_value, draft=None, draft_files=None, files_locked=True):
     # for unpublished records we fallback to the react component so users can change
     # communities
     community_use_jinja_header = bool(community_theme)
-
+    dashboard_route = current_app.config["APP_RDM_USER_DASHBOARD_ROUTES"]["uploads"]
```

Comment:
```suggestion
    dashboard_uploads_route = current_app.config["APP_RDM_USER_DASHBOARD_ROUTES"]["uploads"]
```
or
```suggestion
    dashboard_routes = current_app.config["APP_RDM_USER_DASHBOARD_ROUTES"]
```

what do you think? The `uploads` is not the only dashboard route at the moment.

Reply:
yes you're right, will add that!